src/api/system_api.js

```diff
@@ -115,6 +115,20 @@ module.exports = {
             }
         },
 
+        rotate_root_key: {
+            method: 'PUT',
+            params: {
+                type: 'object',
+                required: ['new_secret'],
```

Comment:
using the secret term and not keeping the root_key term may be a bit confusing, don't you think?

Reply:
fixed to new_root_key